.travis.yml

```diff
@@ -15,14 +15,14 @@ matrix:
   include:
     - stage: Deploy iOS 🍏
       os: osx
-      osx_image: xcode10.1
+      osx_image: xcode12.5
       language: objective-c
       node_js: false
 
       before_install:
-        - nvm install 8.15.0 # enforce using same version as local builds
-        - npm i -g npm@6.4.1 # enforce using same version as local builds
-        - nvm alias default 8.15.0 # otherwise sentry-cli will fail with a cryptic error
+        - nvm install 12.22.1 # enforce using same version as local builds
```

Comment:
We should be using current LTS, 14, if possible.

Reply:
Fixed here 885cd270789b0201c8e8f31b6aeba107aea9b3c0